shell/platform/common/cpp/public/flutter_texture_registrar.h

```diff
@@ -36,6 +38,17 @@ typedef struct {
   size_t height;
 } FlutterDesktopPixelBuffer;
 
+// An image buffer object.
+typedef struct {
+  // The pixel data buffer.
+  const void* buffer;
+  // Width of the pixel buffer.
+  size_t width;
+  // Height of the pixel buffer.
+  size_t height;
+} FlutterDesktopGpuBuffer;
```

Comment:
What's the difference from ``FlutterDesktopPixelBuffer``? If there is no difference... How about changing the name of ``FlutterDesktopPixelBuffer``?

Reply:
> const uint8_t* buffer; // it's an array.

> const void* buffer; // it's an pointer

I roughly understand what you want to mean.. :thinking: 
And this is what I wanted to say at first place:
When we use void* at here, It doesn't indicate what it contains. So  when we try to use it, we have to convert it once.
And we've already started to change the original part.. :heavy_plus_sign:  There is no difference between ``FlutterDesktopPixelBuffer`` and ``FlutterDesktopGpuBuffer``. (I mean Its size or what it contains)

Suggestion) Make something common and use it on both sides. Of course, the original part will need a little modification.
```c++
typedef struct {
  // The pixel data buffer.
  const void* buffer;
  // Width of the pixel buffer.
  size_t width;
  // Height of the pixel buffer.
  size_t height;
} FlutterDesktopBuffer;
```

Or we can put more different information in ``FlutterDesktopGpuBuffer`` and take it to a different structure clearly.

It is just my opinion. :sweat_smile:

Reply:
Thank you for your suggestion, It helped me a lot.
1. If we add const void* buffer in FlutterDesktopPixelBuffer, I think it will make developer confused who use this api.
2. If we use FlutterDesktopGpuBuffer, we need make it more clearly.